pkg/landscaper/controllers/execution/controller.go

```diff
@@ -439,33 +458,35 @@ func HandleAnnotationsAndGeneration(ctx context.Context, log logging.Logger, c c
 		// - force-reconcile annotation
 		// - outdated generation
 		opAnn := lsv1alpha1helper.GetOperation(exec.ObjectMeta)
-		log.Debug("reconcile required, setting observed generation and phase", "operationAnnotation", opAnn, "observedGeneration", exec.Status.ObservedGeneration, "generation", exec.Generation)
+		logger.Debug("reconcile required, setting observed generation and phase", "operationAnnotation", opAnn, "observedGeneration", exec.Status.ObservedGeneration, "generation", exec.Generation)
```

Comment:
should be introduce constants for the keys and add them to the constants file in the logging package?